App/Layout/NavLayout.razor

```diff
@@ -414,6 +444,19 @@
         await InvokeAsync(StateHasChanged);
     }
 
+    public async Task OnPacket(IPacket packet)
+    {
+        if (packet is not AbstractMessageS2CPacket messagePacket) return;
+
+        Console.WriteLine("packet");
```

Comment:
Debug message left in

Reply:
i have no idea what you're talking about, i would never